src/main.rs

```diff
@@ -11,7 +11,7 @@ use kakarot_rpc::eth_rpc::rpc::KakarotRpcModuleBuilder;
 use kakarot_rpc::eth_rpc::run_server;
 use mongodb::options::{DatabaseOptions, ReadConcern, WriteConcern};
 use starknet::providers::jsonrpc::HttpTransport;
-use starknet::providers::{JsonRpcClient, SequencerGatewayProvider};
+use starknet::providers::{JsonRpcClient, Provider as _, SequencerGatewayProvider};
```

Comment:
why `Provider as _`
ive never seen this

Reply:
I like to add this when I am importing a trait but not doing anything with it (just importing for access to the methods).